jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -235,7 +235,7 @@ public Conference(Videobridge videobridge,
                 {
                     // TODO: we can avoid reaching into Videobridge here by merging VideobridgeShim into ConferenceShim
                     long start = System.currentTimeMillis();
-                    IQ response = videobridge.handleColibriConferenceIQ(this, request.getRequest());
+                    IQ response = getShim().handleColibriConferenceIQ(request.getRequest());
```

Comment:
Should this queue live in the `ConferenceShim`?  Ideally the more Colibri/xmpp logic we can move out of the core classes (Videobridge.java, Conference.java) the better.